docs/user-guide/persistent-volumes/index.md

```diff
@@ -356,6 +356,16 @@ parameters:
 Storage classes have a provisioner that determines what volume plugin is used
 for provisioning PVs. This field must be specified.
 
+Administrators are not restricted to specifying the "internal" provisioners
+listed here (whose names are prefixed with "kubernetes.io" and shipped
+alongside Kubernetes), they can also run and specify external provisioners
+which are independent programs that follow a [specification](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/volume-provisioning.md)
+defined by Kubernetes. Authors of external provisioners have full discretion
+over where their code lives, how they're shipped, how they need to be run, what
```

Comment:
"how they're shipped" -> "how the provisioner is shipped, how it needs to be run, what volume plugin it uses, etc."